src/test.py

```diff
@@ -53,6 +53,9 @@ def work():
 
 if __name__ == "__main__":
     if len(sys.argv) > 1 and sys.argv[1] == "--run-tests":
+        # Imitate a reboot of the system
+        print("Restarting Docker", flush=True)
```

Comment:
Can we put this behind an extra flag (that might drag in docopt) because restarting docker is pretty antisocial :)

Reply:
Good idea.